src/lib/node-oauth-client-provider.ts

```diff
@@ -167,6 +169,10 @@ export class NodeOAuthClientProvider implements OAuthClientProvider {
    * @param authorizationUrl The URL to redirect to
    */
   async redirectToAuthorization(authorizationUrl: URL): Promise<void> {
+    if (this.authorizeResource) {
+      authorizationUrl.searchParams.set('resource', this.authorizeResource)
```

Comment:
My test server is Auth0 and the parameter they use is `audience`. It would be nice to have a flag or env var for this to make testing easier but it's out of spec so I figured I'd check before trying to shoehorn it in!

Reply:
Ah, send a PR imo. It'd be nice to have a way to work with all the upstream providers